WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -214,7 +233,7 @@ private extension ItemListView {
         default:
             PointOfSaleItemListErrorView(error: errorState, onAction: {
                 Task {
-                    await posModel.loadItems(base: .root(.products))
+                    await posModel.loadItems(base: .root(.products(search: false)))
```

Comment:
Depending on `selectedItemType` we need to `loadItems` here for `products` or `coupons`.

This worked only because we used `currentController` within `posModel` so we didn't catch it when testing errors for coupons.

Reply:
Done in db7b1be